src/odyssey.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) AcademyJS and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Command } from "commander";
+import chalk from "chalk";
+import mongoose from "mongoose";
+import fs from "fs";
+import dotenv from "dotenv";
+import lodash from "lodash";
+import { SyncConfiguration } from "./types";
+import { TestSuite } from "./models";
+
+dotenv.config();
+const { DATABASE_URL } = process.env;
+
+const syncExercises = async (
+    configuration: SyncConfiguration
+): Promise<void> => {
+    const file = configuration.file ?? "meta.json";
+    const rawData = await fs.promises.readFile(file);
+    const data = JSON.parse(String(rawData)).suites[0].suites.map((suite) => ({
+        title: suite.title,
+        description: suite.description,
+        handle: suite.handle,
+        tests: suite.tests.map((test) => ({
+            title: test.title,
+            description: test.description,
+        })),
+        tags: suite.tags
+    }));
+    const handles = data.map((item) => item.handle);
+
+    if (lodash.uniq(handles).length !== handles.length) {
+        console.log("You have duplicate handles. Terminating sync...");

```

Comment:
Can you make use of chalk here?

```typescript
console.log(`${chalk.redBright("[error]")} Detected duplicate handles.`);
```

Also, it would be helpful if you could print out all the duplicate handles, so we know what to fix.